java/client/src/org/openqa/selenium/remote/ErrorCodes.java

```diff
@@ -74,6 +75,7 @@
   public static final int ELEMENT_NOT_VISIBLE = 11;
   public static final int INVALID_ELEMENT_STATE = 12;
   public static final int UNHANDLED_ERROR = 13;
+  public static final int INTERCEPTING_ELEMENT_ERROR = 14;
```

Comment:
I honestly cannot remember any particular reason for 14. I think it may have just been a gap in the existing numbers and I went for it. I can change it to something else if wanted.